README.md

```diff
@@ -1,2 +1,4 @@
 # yank-restraints
-Benchmarking project testing different restraints schemes for free energy calculations with Yank
+## Benchmarking project testing different restraints schemes for free energy calculations with YANK
+
+This repository houses all the files for the restraints project. Here you will find all the files needed to run YANK simulations on host-guest (cyclodextrin-cyclopentanol) and T4 lysozyme (T4 lysozyme L99A-toluene) complexes.
```

Comment:
Maybe "the restraints project" -> "our project benchmarking the use of restraints binding free energy calculations with Yank, as detailed in our outline" and make "outline" be a hyperlink to the Google Doc (which shouldn't work for anyone unless we've shared it with them).